src/codegen/templates/library_interpreter_templates/default_c_function_call.py.mako

```diff
@@ -0,0 +1,25 @@
+<%page args="function"/>\
+<%
+    from codegen.utilities.interpreter_helpers import generate_interpreter_function_call_args
+    from codegen.utilities.function_helpers import get_arguments_type
+    from codegen.utilities.text_wrappers import wrap, docstring_wrap
+%>\
+
+        cfunc = lib_importer.${'windll' if function.calling_convention == 'StdCall' else 'cdll'}.DAQmx${function.c_function_name}
+\
+## Create argument ctypes types list.
+    %if function.calling_convention == 'StdCall':
```

Comment:
I see code for handling `Cdecl` but this if-statement skips those functions.

I guess they will have different locking, though. Instead of using double-checked locking to lazily initialize `cfunc.argtypes`, they always initialize it and they hold the lock while calling into the DAQmx C API.